tetris.sh

```diff
@@ -17,12 +17,78 @@
 #
 # Author: Kirill Timofeev <kt97679@gmail.com>
 #
+# Localized Support: Rojen Zaman <rojen@riseup.net> | lang/README.md
+#
 # This program is free software. It comes without any warranty, to the extent
 # permitted by applicable law. You can redistribute it and/or modify it under
 # the terms of the Do What The Fuck You Want To Public License, Version 2, as
 # published by Sam Hocevar. See http://www.wtfpl.net/ for more details.
 
 set -u # non initialized variable is an error
+script_dir=$(cd $(dirname $0) && pwd) # define script location
+
+# print language for showing usable languages
+print_languages() {
+    [ -r "$script_dir/lang" ] || {            
+        echo "language(s) doesn't loaded, please check $script_dir/lang directory.";
+        exit 1;
+    }
+    ls -I README.md -1t $script_dir/lang/ 2> /dev/null | cut -d'.' -f1;     # print languages
+}
+
+# show usage when given -h argument
+usage() {
```

Comment:
What do you think about this implementation:```
usage() {
    local available_languages=$(ls -I README.md $script_dir/lang/ 2> /dev/null | cut -d'.' -f1)
    echo "usage: $0 [-h] [-l language]"
    if [ -z "$available_languages"] ; then
        echo "This script supports localization via language files that can be put into $$script_dir/lang"
        echo "Please see url_to_readme_here for more details"
    else
        echo "available languages:"
        ( echo "english (default)" && echo "$available_languages") | sort
    fi
    exit 0
}

Reply:
thanks a lot! it seems perfect.

Reply:
is it `url_to_readme_here` a web url or not?

Reply:
Sorry, I was not clear. Yes, I mean exactly this url.